src/parser/parseReactElement.js

```diff
@@ -1,14 +1,17 @@
 /* @flow */
 
-import React, { type Element as ReactElement } from 'react';
+import React, { type Element as ReactElement, Fragment } from 'react';
 import type { Options } from './../options';
 import {
   createStringTreeNode,
   createNumberTreeNode,
   createReactElementTreeNode,
+  createReactFragmentTreeNode,
 } from './../tree';
 import type { TreeNode } from './../tree';
 
+const supportFragment = Boolean(Fragment);
```

Comment:
How does this works in previous versions, Fragment would be undefined or? I thought if the export is not present it would throw.

Reply:
I was convinced that it return `undefined`, now you spoke about an error I have a doubt. 

After some test in node:
```js
import { Foo } from 'react';
console.log(Foo); // undefined
```

But I'm not sure this really friendly with the bundler (webpack, rollup...). They could have good reason to throw error. We could try like this and I will check with the PR #242 if this break or not. If it break we could try to add a package like `node-semver` to check the `React.version` value or by releasing a major to force people to use React `17.2` with the new version of lib. Will see 😄